include/swift/AST/Import.h

```diff
@@ -429,8 +429,10 @@ struct alignas(uint64_t) ImportedModule {
   };
 };
 
-struct ImportedModuleDesc {
-  ImportedModule module;
+template<class ModuleInfo>
+struct AttributedImport {
```

Comment:
While you're here, could you add a brief doc comment for this type?

Reply:
I did, I think maybe in the next commit:

```
/// Augments a type representing an import to also include information about the
 /// import's attributes. This is usually used with either \c ImportedModule or
 /// \c UnloadedImportedModule.
```